### PR TITLE
Android: Don't ask for device id permission.

### PR DESCRIPTION
**Pull request by vishwesh3**
Issue #1229

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -155,7 +159,6 @@ repositories {
 
 
 dependencies {
-    compile project(':react-native-notifications')
```
Comment (borisyankov) [inline android/app/build.gradle]: Will removing this be a problem?

Comment (vishwesh3) [reply android/app/build.gradle]: No,

It's already get linked in https://github.com/zulip/zulip-mobile/blob/master/android/app/build.gradle#L171

if we don't remove this line, then `react-native-notification` is linked twice and thus causing this error.

```
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Error: more than one library with package name 'com.wix.reactnativenotifications'
```

Comment (borisyankov) [reply android/app/build.gradle]: Cool. The rn-link is causing this issues. I'll be more careful next time I run it.

**Comment by borisyankov** [comment]
Good job!

**Pull request merged**